cypress_shared/pages/temporary-accommodation/manage/bedspaceEditable.ts

```diff
@@ -2,18 +2,6 @@ import type { NewRoom, UpdateRoom } from '@approved-premises/api'
 import Page from '../../page'
 
 export default abstract class BedspaceEditablePage extends Page {
-  getCharacteristicIdByLabel(label: string, alias: string): void {
```

Comment:
I'm struggling to see what's going on in this commit - the json files that have been introduced don't seem to be used anywhere. Has something gone awry with the commit history?

Reply:
Don't think so? There's no new JSON - the changed files are still used in `factories/referenceData.ts`

Where the JSON is now in sync with the API, a room or premises generated by our factories will by default be valid from the API's point-of-view. We don't have to do the extra step of reading valid characteristic/local authority IDs from the page, as the IDs we get from the factory will be fine

Does that make sense?

Reply:
Hmmmm - I'm not sure I follow still. Could you push this refactoring to a seperate branch for me to review? Might help me see the wood for the trees a bit.

Reply:
In an effort to move this along i've read over the latest changes and it looks good and self contained now. It looks like this change was removed and perhaps the conversation continued on Slack?

I've run it locally and it works 👍 

@pezholio are you happy this comment is resolved?

Reply:
Yep, sorry!